Providers/Modules/NPM/Plugin/plugin/npmd_config_lib.rb

```diff
@@ -325,3 +325,4 @@ def self.GetAgentConfigFromUIConfig(uiXml)
     end
 end
 
+
```

Comment:
Absent in latest version of file at https://github.com/Microsoft/OMS-Agent-for-Linux/pull/293